lib/bamboo/adapters/test_adapter.ex

```diff
@@ -1,6 +1,8 @@
 defmodule BambooSMTP.TestAdapter do
   @moduledoc """
-  Used for testing email delivery. Reimplementation of `Bamboo.TestAdapter` with a response more 
+  Based on `Bamboo.TestAdapter`, this module can be used for testing email delivery.
+  
+  The `deliver/2` function will provide a response that follow the format of a SMTP server raw response.
   consistent with elixir conventions and closer to a real request
```

Comment:
Erf... The Github suggestion didn't remove that one. 😅

Reply:
Should I remove this single line? I'm not sure I understood what you say 😅

Reply:
Yeah, that's what I mean. Just read the sentence and you will see it's a remaining part of the former description. 😅

Reply:
done!